tests/conftest.py

```diff
@@ -24,12 +24,18 @@ def fixture_sample_album() -> str:
     return "MPREb_4pL8gzRtw1p"
 
 
-@pytest.fixture(name="badly_indexed_album")
-def fixture_badly_indexed_album() -> str:
+@pytest.fixture(name="missing_indices_album")
+def fixture_missing_indices_album() -> str:
     """Sean Paul - Dutty Classics Collection"""
     return "MPREb_TPH4WqN5pUo"
 
 
+@pytest.fixture(name="disabled_indices_album")
+def fixture_disabled_indices_album() -> str:
```

Comment:
I applaud the effort but I don't think we need to make every browseId under test a fixture. Just use the id where you need it. If you want to keep it to improve readability, I suggest moving it to the file where it's used -- test_browsing in this case

Reply:
Sure thing. Was confused on the mixed usage of fixtures and inline ids, but I'll keep it in mind for future changes.